src/iblt.h

```diff
@@ -29,6 +29,8 @@ SOFTWARE.
 #include <set>
 #include <vector>
 
+static const size_t VALS_32 = 4294967295;
```

Comment:
remove (replace uses with the standard constant as described in another comment)

Reply:
Done.